.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
You'd typically want to use a more recent version for a new project, unless you have a restriction for some reason. The 2.6 branch is up to 2.6.8 now, though is only in security maintenance mode. The 2.7 and 3.0 branches are both in normal maintenance mode.

Source: https://www.ruby-lang.org/en/downloads/branches/ and https://www.ruby-lang.org/en/downloads/releases/

Reply:
Thanks, updated to 2.7.2.